pinakes/data/email_template.html

```diff
@@ -0,0 +1,590 @@
+      border-color: #b8bbbe;
+      border-style: solid;
+    }
+
+    .rh-m-bordered tr:nth-child(even) {
+      background-color: #fafafa;
+    }
+
+    .rh-color__red {
+      color: #a30000;
+    }
+
+    .rh-color__orange {
+      color: #ec7a08;
+    }
+
+    .rh-color__green {
+      color: #486b00;
+    }
+
+    .rh-m-block {
+      display: block;
+    }
+
+    .rh-url {
+      word-break: break-word;
+    }
+
+    .rh-stack > * + * {
+      margin-top: 10px;
+    }
+
+    .rh-inline {
+      display: flex;
+      align-items: center;
+    }
+
+    .rh-icon {
+      display: inline-block;
+    }
+
+    .rh-inline > * + * {
+      display: inline-block;
+      margin-left: 5px;
+    }
+
+    /* dark mode hacks */
+    [data-ogsc] .rh-masthead__brand,
+    [data-ogsb] .rh-masthead__brand {
+      background-color: #151515 !important;
+    }
+
+    @media only screen and (max-width: 320px) {
+      .rh-masthead__block {
+        width: 100% !important;
+        display: block !important;
+      }
+    }
+
+    @media only screen and (max-width: 480px) {
+      .rh-template-container {
+        max-width: 480px !important;
+        /*@editable*/ width:100% !important;
+      }
+
+      .rh-data-table tbody th {
+        width: auto !important;
+      }
+
+      .rh-data-table th,
+      .rh-data-table td {
+        font-size: 14px;
+      }
+
+      .rh-data-table.rh-m-bordered th,
+      .rh-data-table.rh-m-bordered td {
+        padding: 5px;
+        text-align: left;
+      }
+    }
+
+    @media only screen and (max-width: 600px) {
+      body{
+        width: 100% !important;
+        min-width: 100% !important;
+      }
+
+      .rh-template-container {
+        max-width: 600px !important;
+        /*@editable*/ width:100% !important;
+      }
+
+      .rh-masthead__block {
+        width: 33%;
+        max-width: 200px;
+      }
+    }
+  </style>
+
+  </head>
+
+  <body leftmargin="0" marginwidth="0" topmargin="0" marginheight="0" offset="0" class="rh-body">
+    <center>
+      <table align="center" border="0" cellpadding="0" cellspacing="0" height="100%" width="100%" class="rh-body-table">
+        <tr>
+          <td align="center" valign="top" class="rh-body-cell">
+            <table border="0" cellpadding="0" cellspacing="0" class="rh-template-container">
+
+              <!-- Head -->
+              <tr>
+                <td class="rh-head">
+                  <table class="rh-masthead" role="presentation" align="center" border="0" cellpadding="0" cellspacing="0" width="100%">
+                    <tr>
+                      <!--Red Hat logo-->
+                      <td align="center" class="rh-masthead__brand" bgcolor="#151515">
+                        <a href="https://cloud.redhat.com/" target="_blank" class="rh-masthead__brand-link">
+                          <img src="https://$web_url/ui/catalog/fonts/logo_small.svg" alt="Pinakes logo" width="270" height="82" class="rh-masthead__brand-img" />
```

Comment:
Probably don't want to link to the cloud.redhat.com here, and I'm not sure if that logo URL is correct.